### ISSUE TITLE
Create Pachyderm/Kubernetes architectural diagram.

### ISSUE THREAD

**Issue by svekars** [pachyderm/pachyderm#4770]
As a Kubernetes operator, I need a document that explains how Pachyderm works under the hood, what containers are created inside the pachd, etc, and worker pods, how they work and so on and so forth.

One issue that can be described is here (Pahcyderm worker containers and default resources provisioned): https://github.com/pachyderm/pachyderm/issues/4749

We should add an outline to this issue and update it accordingly.

@pappasilenus @JoeyZwicker

**Comment by smthpickboy** [comment]
Really need this as someone new to Pachyderm. Any update on this?

**Comment by mindthevirt** [comment]
@smthpickboy this will come after #3648

**Comment by echohack** [comment]
#5791 addresses this issue. Any new diagrams should open a new issue or PR.
